kukkuu/tests/utils/jwt_utils.py

```diff
@@ -0,0 +1,80 @@
+import time
+import uuid
+from typing import Optional, TYPE_CHECKING
+
+from django.conf import settings
+from jose import jwt
+
+if TYPE_CHECKING:
+    from users.models import User as UserType
+
+
+TEST_JWT_EXP_TIME_IN_SECONDS = 60
+
+
+def get_epoch_timeframe_for_test_jwt():
+    """Get test JWT valid timeframe as epoch times
+
+    Returns:
+        tuple[int, int]: issued at (epoch), expiration (epoch)
+    """
+    epoch_time = int(time.time())
+    return epoch_time, epoch_time + TEST_JWT_EXP_TIME_IN_SECONDS
+
+
+def generate_symmetric_test_jwt(
+    user: "UserType",
+    shared_secret_for_signature: Optional[str] = None,
+    issuer="https://kukkuu-ui.test.hel.ninja",
+    prefix="bearer",
+):
+    headers = {
+        "alg": "HS256",
+        "typ": "JWT",
+        "kid": "zm1m8hQE6UACBzenpoINymuYRf0mL7Z4m-nVHpvy5Kc",
+    }
+    epoch_time, exp_epoch = get_epoch_timeframe_for_test_jwt()
+    payload = {
+        "iat": epoch_time,
+        "auth_time": epoch_time,
+        "exp": exp_epoch,
+        "jti": uuid.uuid4().__str__(),
+        "iss": issuer,
+        "aud": "kukkuu-api-test",
+        "sub": user.uuid.__str__(),
```

Comment:
Same here, normal `str(x)` works, no need for the `x.__str__()`:
```suggestion
        "sub": str(user.uuid),
```